doc/contributing/documentation_guide.md

```diff
@@ -20,10 +20,10 @@ build directory:
 make html
 ```
 
-Then you can preview your changes by opening
+If you don't have a build directory, follow the [quick start guide](building_ruby.md#label-Quick+start+guide)
+up to step 4. Then you can preview your changes by opening
```

Comment:
The added sentence does not look like it should be added to the next sentence.
Shouldn't it be another paragraph?
```suggestion
If you don't have a build directory, follow the [quick start
guide](building_ruby.md#label-Quick+start+guide) up to step 4.

Then you can preview your changes by opening
```

Reply:
I'm fine with either way.